samples/todolist.md

```diff
@@ -0,0 +1,160 @@
+# Pelorus Samples & Use Cases: Todolist
+
+## Deploying a todolist Sample alongside the Webhook/Generic CommitTime Exporter
+
+* The Pelorus exporter requires the Pelorus git source and branch
+  * Fork [this repo](https://github.com/konveyor/pelorus) to https://github.com/your_org/pelorus
```

Comment:
I don't think we need to fork pelorus for this demo, right?

Reply:
We do not *need* to fork pelorus, I agree.  The previous example had it forked and maybe it gives the reader perhaps a dev a better feel.  I can take it or leave it.

Reply:
@KevinMGranger  I would rather NOT have *this* demo on the stable release, some more complicated demo's perhaps.  I would much rather have this demo ALWAYS working from master which is the intent of matching the demo w/ CI.   I can see future demo / samples that are more involved being linked to the last known good stable release. :)